runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -465,6 +465,10 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          vmInfo._interpreterVTableOffset = TR::Compiler->vm.getInterpreterVTableOffset();
          vmInfo._maxHeapSizeInBytes = TR::Compiler->vm.maxHeapSizeInBytes();
          vmInfo._enableGlobalLockReservation = vmThread->javaVM->enableGlobalLockReservation;
+         fe->getNurserySpaceBounds(&vmInfo._nurserySpaceBoundsBase, &vmInfo._nurserySpaceBoundsTop);
+         vmInfo._lowTenureAddress = (void *)fe->getLowTenureAddress();
```

Comment:
Isn't simpler to store `_lowTenureAddress` as UDATA? This would avoid the cast here and avoid the cast in the `TR_J9ServerVM::getLowTenureAddress()` implementation.